master/getting-started/kubernetes/troubleshooting.md

```diff
@@ -36,3 +36,21 @@ depend on how your kubelet is configured.  For deployments using `systemd`, you
 
 The log level can be configured via the CNI network configuration file, by changing the value of the
 key `log_level`.  See [the configuration guide]({{site.baseurl}}/{{page.version}}/reference/cni-plugin/configuration) for more information.
+
+#### How do I configure the Pod IP range? 
+
+When using Calico IPAM, IP addresses are assigned from [IP Pools]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/ippool).
+
+- Calico IPAM assigns IP addresses from IP Pools, by default, all enabled IP Pools are used.
+- You can specify them in the CNI config - this will apply to all containers started in that CNI network. See [CNI config IPAM section]({{site.baseurl}}/{{page.version}}/reference/cni-plugin/configuration#ipam) for more details.
+- You can also specify on a per-pod basis in Kubernetes using an annotation, this overrides the CNI config. See [IPAM Manipulation with Kubernetes Annotations]({{site.baseurl}}/{{page.version}}/reference/cni-plugin/configuration#ipam-manipulation-with-kubernetes-annotations) section in CNI config doc.
+
+#### How do I assign a specific IP address to a pod? 
+
+For most use-cases it's not necessary to assign specific IP addresses to a Kubernetes Pod, and it's recommended to use Kubernetes Services instead.
+However, if you do need to assign a particular address to a Pod, Calico provides two ways of doing this: 
+
+- You can request for an IP that is available in the IPAM, with `cni.projectcalico.org/ipAddrs` annotation.
```

Comment:
Minor wording tweaks  for the first bullet:

- drop the word "for" (e.g. "You can request an IP")
- replace "the" with "Calico" - (e.g. "available in Calico IPAM")
- replace ", with" with "using the" (e.g. "using the `cni.projectcalico.org/ipAddrs`")

All together you get:

"You can request an IP that is available in Calico IPAM using the `cni.projectcalico.org/ipAddrs` annotation.